fv/containers/containers.go

```diff
@@ -191,7 +193,11 @@ func RunWithFixedName(name string, opts RunOpts, args ...string) (c *Container)
 
 	// Prep command to run the container.
 	log.WithField("container", c).Info("About to run container")
-	runArgs := []string{"run", "--name", c.Name}
+	runArgs := []string{"run", "--name", c.Name, "--stop-timeout", fmt.Sprint(opts.StopTimeoutSecs)}
+
+	if opts.StopSignal != "" {
+		runArgs = append(runArgs, "--stop-signal", opts.StopSignal)
+	}
```

Comment:
Are there any cases where it isn't right to use SIGKILL and 0 timeout?

Reply:
I think some of the other repos import this code and use it in other ways so I was scared of changing the default.